dao/OrderDao.py

```
@@ -0,0 +1,267 @@
+                - int: 总订单数量
+        """
+        conn = None
+        cur = None
+        try:
+            conn = self.get_connection()
+            cur = self.get_cursor(conn)
+            
+            offset = (page - 1) * page_size
+            
+            # 使用CTE进行分页查询
+            sql = sql.SQL("""
```

Ruff (0.11.9)

Local variable `sql` referenced before assignment

(F823)